### DISCUSSION TITLE
My CSV files are not loading in gihub pages

### DISCUSSION THREAD

**Discussion by Totocmoi** [community/community#106844] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi, I have a problem: My csv folder is not loading in github pages. There should be a logical explaination, but I don't have it. 
https://github.com/Totocmoi/final_datavis/tree/main
https://totocmoi.github.io/final_datavis/index.html
It says that the csv I request doesn't exist. Most posts about this are people having files named with an underscore, but I don't. All I use is normal csv, with normal js (dans d3.js but I don't think that's the problem here)
Any help would be very apreciated.

**Comment by MattiaValerio** [comment]
Could it be because the files are too large? 
I noticed that your cloud.cvs file is already very large and github doesn't format it.

**Reply by Totocmoi** [reply]
Well wierdly enough, cloud.csv doesn't pose any problem (since my last update that was 20min ago).... I just saw that the only one that is not loading is main-chart.csv (and he's half smaller than cloud.csv)

**Reply by Lanrad23** [reply]
Make sure the way you link to your CSV files in your code is like giving directions from where your HTML or JavaScript file is. For instance, if your CSV files are in a folder named ""csv"" in the same place as your HTML file, you'd write ""./csv/filename.csv"" to point to them. It helped me.

**Comment by SaraRasoulian** [comment, marked as answer]
the path to read your csv files are incorrect

in `main_chart.js` file, update the path to read `main_chart.csv` file, to be like this:
```
d3.csv("datasets/main-chart.csv", function (error, data) {
```

same for `pokemon.csv`:
```
d3.csv("datasets/pokemon.csv", function (error, pokemonData) {
```

**Reply by Totocmoi** [reply]
Thanks a lot ! Since it worked on replit I thought the path was correct... I find it strage that the path is from the html and not the js. In any case, you just saved my project, so thanks you very much !
